examples/main.tf

```diff
@@ -1,12 +1,11 @@
-data "aws_kms_key" "sns_alias" {
```

Comment:
in main.tf examples can you give examples instead of variables? such as [oauth api](https://github.com/variant-inc/terraform-okta-oauth-api-service/blob/master/examples/oath_api_service/main.tf) and [s3](https://github.com/variant-inc/terraform-aws-s3/blob/master/examples/basic/main.tf). Also can you provide testing instructions.